cscs-checks/tools/profiling_and_debugging/berkeleylab-roofline.py

```diff
@@ -0,0 +1,116 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class ErtTestBase(rfm.RegressionTest):
+    """
+    The Empirical Roofline Tool, ERT, automatically generates roofline data.
+    https://bitbucket.org/berkeleylab/cs-roofline-toolkit/
+    """
+    def __init__(self):
+        super().__init__()
+        self.descr = 'Empirical Roofline Toolkit'
+        self.sourcesdir = os.path.join(self.current_system.resourcesdir,
+                                       'roofline', 'ert')
+        self.build_system = 'SingleSource'
+        self.sourcepath = 'kernel1.c driver1.c'
+        self.executable = 'ert.exe'
+        self.build_system.ldflags = ['-O3 -fopenmp']
+        self.sourcesdir = os.path.join(self.current_system.resourcesdir,
+                                       'roofline', 'ert')
+        self.rpt = '%s.rpt' % self.executable
+        self.maintainers = ['JG']
+        self.tags = {'scs'}
+
+    def setup(self, partition, environ, **job_opts):
+        super().setup(partition, environ, **job_opts)
+        if self.num_tasks != 36:
+            self.job.launcher.options = ['--cpu-bind=verbose,none']
+
+
+@rfm.parameterized_test(
+    *[[mpitask, flop]
+    for mpitask in [36, 18, 12, 9, 6, 4, 3, 2, 1]
+    for flop in [1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024]])
+class ErtBroadwellTest(ErtTestBase):
+    def __init__(self, mpitask, flop):
```

Comment:
`mpitask` is a bit confusing as a name. I'd suggest to use either `num_tasks` or `num_ranks`.